lib/widgets/compose_box.dart

```diff
@@ -544,18 +544,15 @@ class _StreamContentInput extends StatefulWidget {
 }
 
 class _StreamContentInputState extends State<_StreamContentInput> {
-  late String _topicTextNormalized;
-
   void _topicChanged() {
     setState(() {
-      _topicTextNormalized = widget.controller.topic.textNormalized;
+      // The relevant state lives on widget.controller.topic itself.
     });
   }
```

Comment:
nit: while we are on this, the lifecycle methods (`initState` and etc.) should probably go before `_topicChanged`

Reply:
Yeah, I think either way is fine. It's a helper method used by all the lifecycle methods, so either before or after them are reasonable homes.